src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs

```diff
@@ -6,13 +6,13 @@ namespace Tizen.NUI
 {
     internal static partial class Interop
     {
-        internal static partial class FrameCallbackInterface
+        internal static partial class FrameUpdateCallbackInterface
         {
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_FrameCallbackInterface")]
-            public static extern global::System.IntPtr new_FrameCallbackInterface();
+            public static extern global::System.IntPtr new_FrameUpdateCallbackInterface();
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_FrameCallbackInterface_director_connect")]
-            public static extern void FrameCallbackInterface_director_connect(global::System.Runtime.InteropServices.HandleRef jarg1, Tizen.NUI.FrameCallbackInterface.SwigDelegateFrameCallbackInterface delegate0);
+            public static extern void FrameUpdateCallbackInterface_director_connect(global::System.Runtime.InteropServices.HandleRef jarg1, Tizen.NUI.FrameUpdateCallbackInterface.SwigDelegateFrameUpdateCallbackInterface delegate0);
```

Comment:
if possible, could you remove this "_" in name
and also if possible, please remove "Swig" word itself~

Reply:
Thank you I fixed it.